spacy/tests/test_cli.py

```diff
@@ -651,6 +650,26 @@ def test_init_config(lang, pipeline, optimize, pretraining):
     load_model_from_config(config, auto_fill=True)
 
 
+@pytest.mark.parametrize("pipeline", [["llm"]])
+@pytest.mark.parametrize("llm_model", ["noop"])
+@pytest.mark.parametrize("llm_task", ["ner", "sentiment"])
+def test_init_config_llm(pipeline, llm_model, llm_task):
```

Comment:
I think this test should probably be in `spacy-llm` unless it's installed by default (similar to `spacy-transformers`, where I'm not sure this is tested at all, but where it could be tested).

Reply:
IMO we should consider installing `spacy-llm` by default going forward. At some point we want to move it to the core codebase, but having it installed by default + kept as separate repo has the advantages that we can still release updates more quickly with `spacy-llm` while not troubling users with an additional install command. The [requirements](https://github.com/explosion/spacy-llm/blob/main/requirements.txt) are minimal.